libs/clipboard/ng-package.json

```diff
@@ -3,6 +3,5 @@
   "dest": "../../dist/libs/clipboard",
   "lib": {
     "entryFile": "src/index.ts"
-  },
-  "allowedNonPeerDependencies": ["clipboard-polyfill"]
+  }
```

Comment:
We can safely remove `clipboard-polyfill` since the native `navigator.clipboard.copyText` is supported in all modern browsers.